draft-bclp-green-terminology.md

```diff
@@ -173,17 +173,27 @@ The proposed metric for router and Ethernet switches is
 
 Where Ti is weighted throughput, Pw is weighted power (energy consumption rate)
 
+                        Ti = a*Tu1 + b*Tu2 + c*Tu3
+                        Pw = a*Pu1 + b*Pu2 + c*Pu3
+
+where (a, b, c) is the relative weight at different usage percentage, a+b+c=1
+
+(Pu1, Pu2, Pu3) is the power at different usage percentage
+
+(Tu1, Tu2, Tu3) is the throughput at different usage percentage
+
 Metric for small network devices:
 
 The metrics adopted for small networking devices intended for home/domestic or small office use is
 
- EER = (0.35Tidle+0.5Tlowpower+0.15TMaximum)/(0.35Pidle+0.5Plowpower_0.15Pmaximum) (Mbit/s/W)
+                        EER = (0.35T_idle+0.5T_lowpower+0.15T_Maximum)/(0.35P_idle+0.5P_lowpower+0.15P_maximum) [Mbit/s/W]
 
-Where throughput is maximum non drop data rate beween wide area Network and
-local area network kport in the ingress direction;
+Where throughput is maximum non-drop data rate beween Wide Area Network and
+Local Area Network ingress port;
 Line rate/speed is maximum possible number of transmitted/received bits.
 Power shall be averaged over 5 minutes, taking measurements every 30 seconds.
-During idle power,IP pings shall be sent via the user interface.
+During idle power, IP pings shall be sent via the user interface.
```

Comment:
```suggestion
During idle power, IP pings shall be sent via the user interface.
```

Why this is mentioned here?

Reply:
I also suggest to remove this sentence since it is not mandatory to have this context information.

Reply:
removed for out of context